### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by XinjieTan**
For Gogle-Code in 2014
Username: xinjie
Thank You!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
-[![Build Status](https://travis-ci.org/fossasia/fossasia.github.io.svg?branch=master)](https://travis-ci.org/fossasia/fossasia.github.io)
```
Comment (samarsault) [inline README.md:1]: @XinjieTan why did you remove the build status image ?

Comment (XinjieTan) [inline README.md:1]: @samarjeet27  I'm sorry I didn't realize! I'll add it back!
